packages/material-ui/src/Badge/Badge.js

```diff
@@ -317,6 +342,10 @@ Badge.propTypes = {
    * @default false
    */
   showZero: PropTypes.bool,
+  /**
+   * The system prop that allows defining system overrides as well as additional CSS styles.
```

Comment:
A note for later, I think that it would be great to add a link to the system page of the documentation as we do for the `classes`  prop: 

<img width="854" alt="Capture d’écran 2020-12-07 à 16 15 49" src="https://user-images.githubusercontent.com/3165635/101368770-d63dcb80-38a7-11eb-859e-81af60bd99cb.png">

It should probably done in the documentation generator to be done once.

Reply:
Makes sense, we can add it, will leave the comment open for visibility